x-pack/plugins/security_solution/public/common/hooks/use_app_toasts.ts

```diff
@@ -239,3 +243,8 @@ export const isEmptyObjectWhenStringified = (item: unknown): boolean => {
     return false;
   }
 };
+
+function postprocessErrorString(str: string): string {
+  // Remove the `[request body]` prefix added by Zod for request validation errors
+  return str.replace(/\[request body\]:/g, '');
+}
```

Comment:
Why do we need to remove the `[request body]` part of the string? How can we distinguish errors in the `body` from errors in the `query` part of the request?